src/displayapp/widgets/PageIndicator.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include <lvgl/lvgl.h>
+
+namespace Pinetime {
+  namespace Applications {
+    namespace Widgets {
+      class PageIndicator {
+      public:
+        PageIndicator(int nCurrentScreen, int nScreens);
+        void Create();
+
+      private:
+        int nCurrentScreen;
```

Comment:
Maybe use `uint8_t` instead of 32 bits integers ? That's not a lot, but we would spare a few bytes in RAM :)